validator/src/main/java/com/amazon/aoc/fileconfigs/ExpectedTrace.java

```diff
@@ -26,7 +26,11 @@
   XRAY_SDK_HTTP_EXPECTED_TRACE("/expected-data-template/xraySDKexpectedHTTPTrace.mustache"),
   SPARK_SDK_HTTP_EXPECTED_TRACE("/expected-data-template/spark/sparkAppExpectedHTTPTrace.mustache"),
   SPARK_SDK_AWSSDK_EXPECTED_TRACE(
-          "/expected-data-template/spark/sparkAppExpectedAWSSDKTrace.mustache")
+          "/expected-data-template/spark/sparkAppExpectedAWSSDKTrace.mustache"),
+  SPRINGBOOT_SDK_HTTP_EXPECTED_TRACE(
```

Comment:
what's the difference between the spark template and the springboot template?

Reply:
the segment hierarchy diff